packages/@aws-cdk/aws-kinesisfirehose/README.md

```diff
@@ -225,6 +225,24 @@ new Alarm(this, 'Alarm', {
 See: [Monitoring Using CloudWatch Metrics](https://docs.aws.amazon.com/firehose/latest/dev/monitoring-with-cloudwatch-metrics.html)
 in the *Kinesis Data Firehose Developer Guide*.
 
+## Compression
+
+Your data can automatically be compressed when it is delivered to S3 as either a final or
```

Comment:
If the user specifies a backup option, does the compression apply to both the backup and the final destination? If so, the "either" in the documentation may be confusing.

Reply:
The backup option will have it's own field for compression. They are not coupled to each other. This should be clear once I have the PR with the backup implementation ready. Will ping you with that, when it's ready.

Reply:
Take a look at the backup PR, and let me know if you still find this unclear: https://github.com/aws/aws-cdk/pull/15725/files